pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -270,6 +270,42 @@ def test_to_datetime_utc_is_true(self):
         expected = pd.DatetimeIndex(data=date_range)
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.parametrize("init_constructor, end_constructor, test_method",
+                             [(Index, DatetimeIndex, tm.assert_index_equal),
+                              (list, DatetimeIndex, tm.assert_index_equal),
+                              (np.array, DatetimeIndex, tm.assert_index_equal),
+                              (Series, Series, tm.assert_series_equal)])
+    def test_to_datetime_utc_true_with_constructors(self,
```

Comment:
there are a couple of tests right above this that are effectively duplicated by this comprehensive test, you can remove then (they test np.array and Index), also add the scalar case